packages/app-transfer/src/md/basic.md

```diff
@@ -0,0 +1,22 @@
+# Transfer
+
+This tab allows you to send funds from one account you own to another account.
+
+# Minimum allowed balance
+
+Accounts with a balance lower than the minimal amount, 100 mili Unit as of writing, are considered as inexistent for the network. If an account's balance ever drops below this amount, it is removed from the network. In this application, it will still be visible, but with a balance of 0.
```

Comment:
```suggestion
Accounts with a balance lower than the minimal amount, 100 miliDOTs as of writing, are considered as inexistent for the network. If an account's balance ever drops below this amount, it is removed from the network. In this application, it will still be visible, but with a balance of 0.
```